.github/workflows/cd-pipeline.yml

```diff
@@ -142,7 +142,10 @@ jobs:
             git reset --hard $(git commit-tree origin/index^{tree} -m "Update index.yaml")
 
             # Rebuild index
-            helm repo index --url https://charts.bitnami.com/bitnami --merge bitnami/index.yaml ../download
+            helm repo index --url oci://registry-1.docker.io/bitnamicharts/ --merge bitnami/index.yaml ../download
+            # Replace .tgz in URL with OCI tag
+            sed -i "s|oci://registry-1.docker.io/bitnamicharts/$app-$chart_version.tgz|oci://registry-1.docker.io/bitnamicharts/$app:$chart_version|" index.yaml
```

Comment:
Can we avoid using "in-place" substitution as we do in our `replace_in_file` bash helper?

Reply:
It is not the first time the `sed -i` is used in the workflows.

In the ci-pipeline it is used several times:

https://github.com/bitnami/charts/blob/b713d56c29f99fa111d079c300fb1e1880be8553/.github/workflows/ci-pipeline.yml#L148
https://github.com/bitnami/charts/blob/b713d56c29f99fa111d079c300fb1e1880be8553/.github/workflows/ci-pipeline.yml#L156
https://github.com/bitnami/charts/blob/b713d56c29f99fa111d079c300fb1e1880be8553/.github/workflows/ci-pipeline.yml#L281